src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+                
```

Comment:
What's up with the whitespace changes in this file?

Reply:
Constantly happens when I use the vs resx editor. No clue why.